src/interfaces/stateItem.ts

```diff
@@ -1,7 +1,7 @@
-interface StateItem {
-  readonly loading?: boolean
-  readonly request?: any
-  readonly data?: any
+interface StateItem<t> {
+  readonly loading: boolean
+  readonly request: { [key: string]: any }
+  readonly data: { [key: string]: t }
   readonly error?: any
```

Comment:
for `error` the TSC knows `Error`. Or you could use a union-type `Error|string|undefined` etc...

Reply:
The type of `StateItem.error` had not been updated in this PR, but I changed it to `Error | string` to address this feedback.